source/_posts/angular-session-storage.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Session Storage in Angular
+contributor: A M Sanjeev
+avatar: a-m-sanjeev.jpg
+description: Learn how to use Session Storage Angular.
+date: 2023-03-17
+tags: [angular]
+id: angular-session-storage
+---
+
+## Table of Contents
+<!-- toc -->
+
+## Introduction
+
+Local storage, session storage, and cookies are all terms that most of us are familiar with. But what are they exactly, what problems do they solve, and how do they differ? This article will explain what problems they solve, how they differ, as well as how to use `sessionStorage` in a simple angular project.

```

Comment:
`Angular` is the framework. It should start with uppercase. Please fix all such places.

Reply:
earlier you said angular should be lowercase. please recommend only one thing.